dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/zk/ZKServer.java

```diff
@@ -121,7 +140,7 @@ private static synchronized void startLocalZkServer(final int port, final String
             logger.info("Zookeeper server started ");
             isStarted.compareAndSet(false, true);
 
-            zkServer.initializeAndRun(args);
+            zooKeeperServerMain.initializeAndRun(args);
         } catch (QuorumPeerConfig.ConfigException e) {
             logger.warn("Caught exception while starting ZK", e);
         } catch (IOException e) {
```

Comment:
```suggestion
            zooKeeperServerMain.initializeAndRun(args);
        } catch (QuorumPeerConfig.ConfigException e) {
            logger.warn("Caught exception while starting ZK", e);
        } catch (IOException e) {
```
Hi,

Is it better to merge tow exceptions into one catch block if the two exceptions are handled in the same way?
If it is convenient for you, I think you can modify this part.
Thx a lot for your contribution.

Reply:
solved